cmd/provider-list/converter.go

```diff
@@ -72,3 +82,72 @@ func getProviderAndServiceName(name string) []string {
 	}
 	return nil
 }
+
+type ConverterParameters struct {
+	regorg        string
+	familyVersion string
+	monolith      string
+	refPackage    string
+}
+
+func (cc *ConverterParameters) ConfigurationMetadataV1(c *xpmetav1.Configuration) error {
+	var convertedList []xpmetav1.Dependency
+
+	for _, provider := range c.Spec.DependsOn {
+		if *provider.Provider == fmt.Sprintf("xpkg.upbound.io/upbound/%s", cc.monolith) {
+			continue
+		}
+		convertedList = append(convertedList, provider)
+	}
+
+	for ssopName := range SSOPNames {
```

Comment:
We need to be careful here for the list of `ssop`s (family providers). If a Kubernetes source is also configured, we will also get ssops for MRs that don't exist in any of the compositions. We should only be adding provider dependencies that are imposed by the compositions.